incubator/group/keeper_test.go

```diff
@@ -194,18 +198,163 @@ func TestCreateGroupAccount(t *testing.T) {
 	}
 }
 
+func TestCreateProposal(t *testing.T) {
+	amino := codec.New()
+	pKey, pTKey := sdk.NewKVStoreKey(params.StoreKey), sdk.NewTransientStoreKey(params.TStoreKey)
+	paramSpace := subspace.NewSubspace(amino, pKey, pTKey, group.DefaultParamspace)
+
+	groupKey := sdk.NewKVStoreKey(group.StoreKeyName)
+	k := group.NewGroupKeeper(groupKey, paramSpace, baseapp.NewRouter(), &testdata.MyAppProposal{})
+	blockTime := time.Now()
+	ctx := group.NewContext(pKey, pTKey, groupKey).WithBlockTime(blockTime)
+	defaultParams := group.DefaultParams()
+	paramSpace.SetParamSet(ctx, &defaultParams)
+
+	members := []group.Member{{
+		Address: []byte("valid-member-address"),
```

Comment:
Would it be consistent if we use double dash `--` here
```suggestion
		Address: []byte("valid--member-address"),
```

Reply:
🤔 Aaron had run into this as well. The address must have a length of `20` bytes to be valid. I used `-` to fill the gaps.

Reply:
Understand, shouldn't be an issue.